.appveyor.yml

```diff
@@ -1,36 +0,0 @@
-os: Visual Studio 2019
```

Comment:
@o01eg is there any reason we need the AppVeyor builds still?

Reply:
Not sure, maybe we left it as alternative to Github Actions